tensorboard/webapp/metrics/store/metrics_store_internal_utils_test.ts

```diff
@@ -639,4 +645,84 @@ describe('metrics store utils', () => {
       expect(nextCardStepIndexMap).toEqual({card1: 1});
     });
   });
+
+  describe('getImageCardStepValues', () => {
+    const cardId = 'test-card-id';
+    const cardMetadataMap = {
+      [cardId]: {
+        runId: 'test run Id',
+        plugin: PluginType.IMAGES,
+        tag: 'tagC',
+        sample: 0,
+      },
+    };
+    let loadables: TimeSeriesLoadables = {
+      scalars: {runToLoadState: {}, runToSeries: {}},
+      histograms: {runToLoadState: {}, runToSeries: {}},
+      images: {runToLoadState: {}, runToSeries: {}},
+    };
+    let timeSeriesData: TimeSeriesData = {
+      scalars: {},
+      histograms: {},
+      images: {},
+    };
+
+    beforeEach(() => {
+      loadables = {
```

Comment:
I don't think there is any reason for `loadables` to exist? You can just create inline in timeSeriesData? The extra level of indirection is very confusing for understanding the tests.